README.md

```diff
@@ -22,79 +22,199 @@ This version supports serverless functions on AWS (Lambda functions) only, suppo
+Questions that needs answers:
+- AWS Access Key (Required)
+- AWS Secret key (Required)
+- The AWS region you want FunctionClarity installed in (Required)
+- Default AWS S3 Bucket for code signing storage (Optional if you have only one FunctionClarity instance across all regions). Note that the FunctionClarity S3 bucket name is global i.e if you have a FunctionClarity deployed per region, it is required to give each instance a unique name even if they are in deployed in different regions. Default sytem generated name is "FunctionClarity".
+- Custom function tags you want to filter on (Optional. Blank scans for all functions. Tag example "sm-hello1" (This translates to key: sm-hello1, value:<blank-this-field-is-not-used>)
+- Function regions (Optional. Blank scans all regions in this account. It is best to start off scanning one region and with specific function tags.) **Warning**: Using blank region filters and blank filter tags means FunctionClarity is scanning (and possibly blocking) ALL functions in ALL regions
+- Post verification action (Required)
+- SNS arn for notifications (Optional)
+- Existing CloudTrail instance to use (Optional)
+- Keyless mode (Optional. See documentation on this mode)
+- Path for custom (existing) codesign key (Optional)
+- Password to protect local private key that is created by FunctionClarity (Required)
+
+```
+# ./functionclarity init aws
+```
+Example output:
+```
+enter Access Key: <YOUR_ACCESS_KEY>
+enter Secret Key: <YOUR_SECRET_KEY>
+enter region: us-west-2
+enter default bucket (you can leave empty and a bucket with name functionclarity will be created):
+enter tag keys of functions to include in the verification (leave empty to include all): sm-hello1
+enter the function regions to include in the verification, i.e: us-east-1,us-west-1 (leave empty to include all): us-west-2
+select post verification action : (1) for detect; (2) for block; leave empty for no post verification action to perform: 2
+enter SNS arn if you would like to be notified when signature verification fails, otherwise press enter:
+is there existing trail in CloudTrail (in the region selected above) which you would like to use? (if no, please press enter):
+do you want to work in keyless mode (y/n): n
+enter path to custom public key for code signing? (if you want us to generate key pair, please press enter):
 Enter password for private key:
+Enter password for private key again:
+File cosign.key already exists. Overwrite (y/n)? y
+Private key written to cosign.key
+Public key written to cosign.pub
+Uploading function-clarity function code to s3 bucket, this may take a few minutes
+function-clarity function code upload successfully
+deployment request sent to provider
+waiting for deployment to complete
+deployment finished successfully
+```
+
+## Use FunctionClarity to Sign Your Code
 
+```
+./functionclarity sign aws code src/
+```
+
+Example output:
+```
+# ./functionclarity sign aws code src/
```

Comment:
again - in this example you are showing how to sign the function clarity code and not the example function.